session.go

```diff
@@ -1423,6 +1427,25 @@ func (q *Query) releaseAfterExecution() {
 	q.decRefCount()
 }
 
+// SetKeyspace will enable keyspace flag on the query.
+// It allows to specify the keyspace that the query should be executed in
+//
+// Only available on protocol >= 5.
+func (q *Query) SetKeyspace(keyspace string) *Query {
+	q.keyspace = keyspace
+	return q
+}
+
+// WithNowInSeconds will enable the with now_in_seconds flag on the query.
+// Also, it allows to define now_in_seconds value.
+//
+// Only available on protocol >= 5.
+func (q *Query) WithNowInSeconds(now int) *Query {
+	q.useNowInSeconds = true
+	q.nowInSecondsValue = now
```

Comment:
I wonder if we should just use `*int` on the internal field so we can check if it's `nil` and therefore remove the need of having a separate `useNowInSeconds` flag

Reply:
It sounds like a cool idea! I'm going to change it as well

Reply:
It's done.